doc_source/sam-tutorial.md

```diff
@@ -336,25 +336,33 @@ In this step, you use your local workstation to add a test that AWS CodeStar run
 
    ```
    version: 0.2
-   
+
    phases:
-   
-     install:
-       commands:
-         - pip install pytest
-   
-     pre_build:
-       commands:
-         - pytest
-   
-     build:
-       commands:
-         - aws cloudformation package --template template.yml --s3-bucket $S3_BUCKET --output-template template-export.yml
-   
+      install:
+         runtime-versions:
+            python: 3.7
+
+         commands:
```

Comment:
Hi--
Can you also add the following to the install -> commands:
# Upgrade AWS CLI to the latest version
- pip install --upgrade awscli

That way, we make sure we have an updated AWS CLI.
Thanks!

Reply:
I'll add the command to upgrade the AWS CLI. Thank you for contributing to the AWS CodeStar docs.